### PR TITLE
Continuous Integration process

### PR DESCRIPTION
**Pull request by situx**
I created a CI process with Github Actions which builds a Github page in a separate branch of this repository.
The Github page contains ontology documentation generated by pylode for GeoSPARQL 1.0 and GeoSPARQL 1.1 from the currently available geo.ttl files.
The result looks like this: https://situx.github.io/ogc-geosparql/
While the style can definitely be improved I think it is important to verify the TTL files we post I the repository automatically using CI.
I fixed a missing namespace during this process.
However, I did not manage to create an ontology documentation for all of the ttl files as they require an ontology definition inside. Maybe Nic can help with that?
Would be nice to be able to show the current improvements of the GeoSPARQL ontology on a Github page to anyone who is interested.

### CODE REVIEW

**Comment by nicholascar** [comment]
This is a great idea and yesterday in the W3C's Dataset Exchange WG I got a request for continuous delivery of documented RDF from this SWG (so other groups can follow the work).

As mentioned in the meeting, to document a _profile_, as opposed to an _ontology_, you just need to set an optional switch `-p` in pyLODE to indicate "prof":

```
~$ pylode pylode -i ../../1.0/profile.ttl -o profile.html -p prof
```

for SKOS vocabularies, the "vocpub" profile:

```
pylode -i ../../1.1/funcsrules.ttl -o funcsrules.html -p vocpub
```

**Comment by situx** [comment]
The CI process now creates a PDF and HTML from the given ASCIIDoc, creating pylode documentation for the profile, simple features vocabulary and the functions works as well.

**Comment by situx** [comment]
If there are any suggestions on how to style the page I am looking forward to them

**Review by nicholascar — APPROVED** [review]

**Comment by nicholascar** [comment]
@jabhay we need a second review for this PR, can you do it? We can't too easily check outputs until it's merged (other than re-creating it all in our our personal GitHub accounts) but since this adds new files, why not approve it and then we can PR to touch out outputs once generated.

I think some small HTML cleaning of outputs will be needed, e.g. removing placeholder images.

**Review by jabhay — APPROVED** [review]
Looks good. Let's look at the generated documents and re-assess.

**Pull request merged**
